bin/fts_quickstart

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+echo "Installing Java SDK..."
+# Find our package manager
+if VERB="$( command -v apt-get )" 2> /dev/null; then
+   echo "Debian-based OS detected"
+   sudo apt-get install -y openjdk-8-jdk 2> /dev/null
+elif VERB="$( command -v yum )" 2> /dev/null; then
+   echo "Modern Red Hat-based OS detected"
+   sudo yum install -y java-1.8.0-openjdk.x86_64 2> /dev/null
+else
+   echo "No valid package manager detected; try one of apt-get, yum."
```

Comment:
Seems a bit silly to recommend trying package managers you've already tried.

Reply:
The intent was to recommend to the dev to try installing one of `apt` or `yum` - I can update the message to be clearer.

Reply:
Ah, I see! yes, clearer is better :)